extensions/shared/external-media/editor.scss

```diff
@@ -0,0 +1,363 @@
+		animation: loading-fade 1.6s ease-in-out infinite;
+		background-color: $light-gray-secondary;
+		border: 0;
+	}
+
+	// Toolbar for "insert" and pagination button.
+	.jetpack-external-media-browser__media__toolbar {
+		position: fixed;
+		position: sticky;
+		bottom: 0;
+		left: 0;
+		width: 100%;
+		background: white;
+		padding: 20px 0;
+		display: flex;
+		justify-content: flex-end;
+	}
+
+	.jetpack-external-media-browser__loadmore {
+		clear: both;
+		display: block;
+		margin: 24px auto 48px auto;
+	}
+}
+
+// Show more thumbs beyond mobile.
+@media only screen and ( min-width: 600px ) {
+	.jetpack-external-media-browser .jetpack-external-media-browser__media__item {
+		width: 20%;
+		padding-top: 20%;
+	}
+}
+
+/**
+ * The specific wrappers for Google and Pexels.
+ */
+
+.jetpack-external-media-header__view {
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	margin-bottom: 48px;
+
+	select {
+		max-width: 200px !important;
+	}
+}
+
+.jetpack-external-media-header__filter,
+.jetpack-external-media-header__view {
+	label {
+		margin-right: 10px;
+	}
+
+	.components-base-control {
+		padding-right: $grid-size;
+		margin-bottom: 0;
+	}
+
+	.components-base-control__label {
+		margin-bottom: 0;
+	}
+
+	.components-base-control__field {
+		display: flex;
+		align-items: center;
+		margin-bottom: 0;
+	}
+
+	.components-base-control + .components-base-control {
+		padding-left: $grid-size * 2;
+	}
+}
+
+.jetpack-external-media-header__filter {
+	display: flex;
+	flex-wrap: wrap;
+	align-items: center;
+
+	.jetpack-external-media-googlephotos-filter {
+		display: flex;
+		align-items: center;
+		margin-right: 7px;
+	}
+}
+
+.jetpack-external-media-header__pexels {
+	display: flex;
+	margin-bottom: 48px;
+
+	.components-base-control {
+		flex: 1;
+		margin-right: 12px;
+	}
+
+	.components-base-control__field {
+		margin-bottom: 0;
+	}
+
+	.components-base-control__field,
+	.components-text-control__input {
+		height: 100%;
+	}
+}
+
+/**
+ * Basic Responsiveness
+ */
+
+.jetpack-external-media-placeholder__open-modal {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	padding: 0;
+	position: absolute;
+	right: 0;
+	margin-top: -48px;
+	z-index: 1;
+
+	.components-button {
+		margin: 0;
+		padding: 12px;
+		background: none;
+
+		&::before {
+			content: none;
+		}
+
+		svg {
+			display: block;
+			fill: currentColor;
+		}
+	}
+}
+
+.jetpack-external-media-browsing
+	> div.components-placeholder:not( .jetpack-external-media-replacedholder ) {
+	display: none;
+}
+
+.jetpack-external-media-browser__empty {
+	width: 100%;
+	text-align: center;
+	padding-top: 2em;
+}
+
+.jetpack-external-media-auth {
+	max-width: 340px;
+	margin: 0 auto;
+	text-align: center;
+
+	p {
+		margin: 2em 0;
+	}
+}
+
+.jetpack-external-media-filters {
+	display: flex;
+	justify-content: space-between;
+}
+
+// Reset placeholder button margin.
+.components-placeholder__fieldset .components-dropdown .components-button {
```

Comment:
@retrofox This is not scoped to Jetpack neither and needs fixing.

Reply:
Fixed in #16144.